README.md

```diff
@@ -1,13 +1,36 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was an introduction to Mongo DB and Mongoose. This is the first time we stored our data in a database. Then it was about creating RESTful routes to display the data.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Having imported the data I set up the database in Mongo DB compass. Then the model Movie was created to handle each item in the data array. Different routes were created to show filtered data. Then I added some error handling in case the user entered invalid entries. Models in Mongoose were used for data and also Mongoose methods like find, findOne. Async and await was used to handle asynchronous operations. Path parameters and query parameters were used for the different endpoints.If I had more time I would have created another model and connected them.
```

Comment:
Looks like you're missing a space between the beginning of the last sentence in this paragraph and the previous sentence.